apps/src/pythonlab/layout/ConsoleAndPreviewShare.tsx

```diff
@@ -1,9 +1,36 @@
 import React from 'react';
 
+import MiniAppPreview from '@cdo/apps/codebridge/MiniAppPreview/MiniAppPreview';
 import HorizontalOutput from '@cdo/apps/codebridge/Workspace/HorizontalOutput';
 
-const ConsoleAndPreviewShare: React.FunctionComponent = () => {
-  return <HorizontalOutput height={800} setOutputHeight={() => {}} />;
+interface ConsoleAndPreviewShareProps {
+  consoleVisible: boolean;
+  height: number;
+  width: number;
+}
+
+const ConsoleAndPreviewShare: React.FunctionComponent<
+  ConsoleAndPreviewShareProps
+> = ({consoleVisible, height, width}) => {
+  if (consoleVisible) {
+    return (
+      <HorizontalOutput
+        height={height}
+        width={width}
+        setOutputHeight={() => {}}
+      />
+    );
+  }
+
+  return (
```

Comment:
For follow-up:
This is probably a question for @samantha-code - on mobile, the console is quite narrow. Wondering if we should offer just miniapp preview when mobile?
<img width="503" alt="Screenshot 2025-03-12 at 8 38 59 AM" src="https://github.com/user-attachments/assets/86d0dd2b-7048-4ade-9976-46cb952aaba5" />

<img width="528" alt="Screenshot 2025-03-12 at 8 38 26 AM" src="https://github.com/user-attachments/assets/5785ad35-4400-4c0c-8903-843154595715" />

Reply:
Oh interesting, I'll follow up about mobile share. It wasn't a consideration with this.